WordPress/src/main/java/org/wordpress/android/ui/posts/mediauploadcompletionprocessors/GalleryBlockProcessor.java

```diff
@@ -49,7 +49,7 @@ public GalleryBlockProcessor(String localId, MediaFile mediaFile, String siteUrl
         // if a match is found, proceed with replacement
         if (targetImg != null) {
             // replace attributes
-            targetImg.attr("src", mRemoteUrl);
+            targetImg.attr("src", mRemoteImageUrl);
```

Comment:
If we make the change from [the suggestion here](https://github.com/wordpress-mobile/WordPress-Android/pull/16875/files#r925143768), we could keep this as `mRemoteUrl` (and the other references as well). Wdyt?

Reply:
Sounds good! I've applied the changes to remove `mRemoteImageUrl`